patches/node/win_almost_fix_race_detecting_esrch_in_uv_kill.patch

```diff
@@ -0,0 +1,70 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
This addresses https://github.com/nodejs/node/issues/51766 - which we were seeing in CI on Windows.